google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -325,6 +326,33 @@ void PatchBucketStorageClassWithBuilder(google::cloud::storage::Client client,
   (std::move(client), bucket_name, storage_class);
 }
 
+void GetBucketClassAndLocation(google::cloud::storage::Client client, int& argc,
+                               char* argv[]) {
+  if (argc != 2) {
+    throw Usage{"get-bucket-class-and-location <bucket-name>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  // [START storage_get_bucket_class_and_location]
+  namespace gcs = google::cloud::storage;
+  using ::google::cloud::StatusOr;
+  [](gcs::Client client, std::string bucket_name) {
+    StatusOr<gcs::BucketMetadata> bucket_metadata = client.GetBucketMetadata(
+        bucket_name, gcs::Fields("name,location,storageClass"));
```

Comment:
I'm questioning if Fields should be selected in samples. 
It has additional overhead on the user knowing field names. I understand it's also a best practice to only select the fields you need. The sample should show a user the "happy path".

I'm open to discussing this more. If no objections, please remove the field selection.

Reply:
No objection, removed.